weave-js/src/components/ToggleButtonGroup/ToggleButtonGroup.tsx

```diff
@@ -0,0 +1,65 @@
+import * as ToggleGroup from '@radix-ui/react-toggle-group';
+import React from 'react';
+import {twMerge} from 'tailwind-merge';
+
+import {Button} from '../Button';
+import {IconName} from '../Icon';
+import {Tailwind} from '../Tailwind';
+import {ToggleButtonGroupSize} from './types';
+
+export type ToggleButtonGroupProps = {
+  options: string[];
+  value: string;
+  size: ToggleButtonGroupSize;
+  isDisabled?: boolean;
+  icons?: Array<IconName | undefined>;
+  onValueChange: (value: string) => void;
+};
+
+export const ToggleButtonGroup = React.forwardRef<
+  HTMLDivElement,
+  ToggleButtonGroupProps
+>(({options, value, size, isDisabled = false, icons, onValueChange}, ref) => {
+  if (options.length < 2) {
```

Comment:
should we even render the component if its in this invalid state? maybe we should just return null, and then add a jsdocs comment that this component should only be rendered if `options.length >= 2`

Reply:
i'm not sure about the specific ask for this component though so maybe we do nothing, but generally we should avoid console.error as it would show up on production saas (if some customer inspected the logs they would see this error)